pyproject.toml

```diff
@@ -9,6 +9,8 @@ python = "^3.8"
 Flask = "^2.2.5"
 python-dotenv = "^0.14.0"
 requests = "^2.31.0"
+pytest = "^8.1.1"
+selenium = "^4.18.1"
```

Comment:
Minor, but worth noting Poetry's "group" option for adding dependencies, similar to adding dev dependencies in npm. E.g. we could use `poetry add pytest --group test` to mark it as a test dependency, and then we can later use commands like `poetry install --without test` to limit our dependencies in production etc.

Definitely not required here, but worth considering

Reply:
Thanks for this! I'll keep it in mind for the future